### ISSUE TITLE
Provenance of Disease-Phenotype relations

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1423]
We need better provenance of the Disease-Phenotype relationships.
Which relationships were derived from Orphanet?

https://monarchinitiative.org/disease/DOID%3A14323#phenotypes

cc: @annieolry
@drseb

**Comment by iamkoehler** [comment]
Related to https://github.com/monarch-initiative/dipper/issues/428

I don't know how disease-phenotype data is input into monarch. Where is the data coming from.

**Comment by kshefchek** [comment]
completely agree, but just to answer the question, these come from:
http://compbio.charite.de/jenkins/job/hpo.annotations/lastStableBuild/artifact/misc/phenotype_annotation.tab/*view*/

This is a case where we need to capture transitive provenance better, as in @drseb may be pulling in these annotations from Orphanet but we are not capturing this provenance data in our ingest.

**Comment by iamkoehler** [comment]
Yes. The ones from from Orphanet are marked as TAS and have source "orphanet". Note that Orphanet likes to have "ORPHA:10" instead of "ORPHANET:10". My mistake. Will fix in next release.

Example line:
`ORPHANET	10	48,XXYY SYNDROME		HP:0000023	ORPHANET:10	TAS		HP:0040283		O		2017.02.24	orphanet
`

**Comment by cmungall** [comment]
We use "Orphanet:NNN". This is based on ORDO, which uses purls like http://www.orpha.net/ORDO/Orphanet_10
